### ISSUE TITLE
case-senstive URL

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kipyegonmark/webgoat#45]
```
There are many places in this project that refer to the url for webgoat as

http://localhost/WebGoat/attack

instead of

http://localhost/webgoat/attack

This took me a while to figure out why I was getting 404 by the Apache Tombcat 
server. I suggest you grep for all the places with the url using the wrong case.
```

Original issue reported on code.google.com by `christop...@gmail.com` on 5 Oct 2011 at 7:20

**Comment by GoogleCodeExporter** [comment]
```
Here's one example from the FAQ.

Q. I put the WebGoat war file in my tomcat/webapps directory and the 
http://localhost/WebGoat/attack URL doesn't work.

A. Rename the downloaded war file to WebGoat.war. Delete the existing 
tomcat/webapps/WebGoat directories. Restart Tomcat. 
```

Original comment by `christop...@gmail.com` on 5 Oct 2011 at 8:22

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 5.4 
```

Original comment by `mayhe...@gmail.com` on 23 Apr 2012 at 12:33
- Changed state: **Fixed**
